config.toml

```diff
@@ -0,0 +1,22 @@
+raw_output = ""
+user_token = ""
+
+[filter]
+  contact = ""
+  email = ""
+  limit = "99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999"
+  metadata = ""
+  name = ""
+  offset = ""
+  state = ""
+  status = ""
+  topic = ""
```

Comment:
Let's leave only `limit`, `offset`, and `topic`, but we need to be able to use `topic` for sending messages. Other filters are dynamic and there is no need to put them all.